truss/utils.py

```diff
@@ -48,3 +49,16 @@ def build_truss_target_directory(model_framework_name: str) -> Path:
     )
     target_directory_path.mkdir(parents=True)
     return target_directory_path
+
+
+def get_gpu_memory():
+    # https://stackoverflow.com/questions/59567226/how-to-programmatically-determine-available-gpu-memory-with-tensorflow
+    try:
+        command = "nvidia-smi --query-gpu=memory.used --format=csv"
+        memory_free_info = (
+            sp.check_output(command.split()).decode("ascii").split("\n")[1]
+        )
+        memory_free_values = int(memory_free_info.split()[0])
+        return memory_free_values
+    except Exception:
+        return 0
```

Comment:
It's probably better to return None in this case, to indicate the ability to identify this information, rather than return a valid but incorrect value.